strongarm_msdns/config.py

```diff
@@ -0,0 +1,33 @@
+try:  # Python 2
+    import ConfigParser as configparser
+except ImportError:  # Python 3
+    import configparser
+
+
+CONFIG_FILE = 'strongarm-msdns.ini'
+SECTION = 'strongarm-msdns'
+
+
+def write_config(**kwargs):
+    """
+    Write to disk a configuration file containing the given keyword arguments.
+
+    """
+    config = configparser.ConfigParser()
+    config.add_section(SECTION)
+
+    for key in kwargs:
+        config.set(SECTION, key, kwargs[key])
+
+    with open(CONFIG_FILE, 'w') as fout:
+        config.write(fout)
```

Comment:
I feel like the caller will need to do error handling on this anyway. They can just catch `IOError`, instead of us doing it and reraising as `MicrosoftDnsException`. Not sure which one is more idiomatic.